src/Symfony/Component/Workflow/Workflow.php

```diff
@@ -286,7 +292,7 @@ public function getMetadataStore(): MetadataStoreInterface
         return $this->definition->getMetadataStore();
     }
 
-    private function buildTransitionBlockerListForTransition(object $subject, Marking $marking, Transition $transition): TransitionBlockerList
+    private function buildTransitionBlockerListForTransition(object $subject, Marking $marking, Transition $transition, array $context = []): TransitionBlockerList
```

Comment:
$context is dead code here + it's the same for many other places

Reply:
:+1: